test/fetch/util.js

```diff
@@ -178,6 +199,25 @@ test('setRequestReferrerPolicyOnRedirect', async (t) => {
     strictEqual(request.referrerPolicy, 'origin')
   })
 
+  await t.test('should ignore empty string as policy', (t) => {
+    const request = {
+      referrerPolicy: 'no-referrer, strict-origin-when-cross-origin'
+    }
+
+    const actualResponse = {
+      headersList: new HeadersList()
+    }
+
+    const { strictEqual } = tspl(t, { plan: 1 })
+
+    actualResponse.headersList.append('Connection', 'close')
+    actualResponse.headersList.append('Location', 'https://some-location.com/redirect')
+    actualResponse.headersList.append('Referrer-Policy', 'origin, asdas, asdaw34, no-referrer,,')
+    util.setRequestReferrerPolicyOnRedirect(request, actualResponse)
+
+    strictEqual(request.referrerPolicy, 'no-referrer')
```

Comment:
this test should use an actual server and it shouldn't use internals

Reply:
this is clearly an util test. it is allowed to use internals.

Reply:
This test is completely useless, to the point I'd call it bloat. What is it testing? That internals, under the perfect circumstances, work properly? We need tests that actually use the public api (fetch) to ensure it's working as intended.

Reply:
These tests were good enough for you when you approved the PR #1717 which introduced those tests. Now they are complete useless bloat?

Reply:
that was 2 years ago